plugins/modules/win_firewall.ps1

```diff
@@ -50,7 +55,20 @@ Try {
                 $result.changed = $true
                 $result.$profile.enabled = $true
             }
-
+            if($null -ne $inbound_action) {
+                $inbound_action = (Get-Culture).TextInfo.ToTitleCase(($inbound_action.ToLower() -replace "_", " ")) -replace " ", ""
```

Comment:
You should do `[Globalization.CultureInfo]::InvariantCulture.TextInfo.ToTitleCase(...)` here and below to ensure it always replaces `not_configured` to the proper value. This ensures that regardless of the actual Locale set the operation will be the same which we know to work. Also you can condense the replace into just 1 call

```suggestion
                $inbound_action = [Globalization.CultureInfo]::InvariantCulture.TextInfo.ToTitleCase($inbound_action.ToLower()) -replace '_', ''
```